.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x]
+        node-version: [12.x, 14.x, 16.x]
```

Comment:
Make sure to update the release workflow too, it's still on Node 10: https://github.com/readmeio/markdown/blob/next/.github/workflows/release.yml#L15

Reply:
Oh thanks!